pkg/util/util.go

```diff
@@ -0,0 +1,37 @@
+package util
+
+import (
+	"bufio"
+	"bytes"
+	"io"
+
+	"github.com/pkg/errors"
+)
+
+// MegaByte denotes one megabyte and is intended to be used as a limit in `ReadAtMost`.
+const MegaByte = 1 << 20
```

Comment:
(It feels weird to have this definition in an, overall, pretty random package. It’s not really a concept specific to this :)

OTOH it works just fine.)